snapcraft/plugins/kbuild.py

```diff
@@ -122,6 +122,21 @@ def __init__(self, name, options, project):
         if logger.isEnabledFor(logging.DEBUG):
             self.make_cmd.append('V=1')
 
+    def enable_cross_compilation(self):
+        logger.info('Cross compiling kernel target {!r}'.format(
```

Comment:
It's still the kernel architecture...
Alternatively we can drop it entirely and just rely on the regular message that refers to the debian target architecture.